pgsync/settings.py

```diff
@@ -106,6 +106,8 @@
 REDIS_AUTH = env.str("REDIS_AUTH", default=None)
 # number of items to read from Redis at a time
 REDIS_CHUNK_SIZE = env.int("REDIS_CHUNK_SIZE", default=1000)
+# number of items to write to Redis at a time
+REDIS_WRITE_CHUNK_SIZE = env.int("REDIS_WRITE_CHUNK_SIZE", default=1000)
```

Comment:
I think eventually the other variable should be renamed to REDIS_READ_CHUNK_SIZE